### ISSUE TITLE
List of records add empty line at each list item

### ISSUE THREAD

**Issue by RDBModel** [MaybeJustJames/yaml#29]
Another case that, perhaps, can be improved https://ellie-app.com/k5NYB5sL7zDa1
![image](https://user-images.githubusercontent.com/100466382/200191616-d527a818-0c56-4eaa-a259-4939fb6da702.png)
If there is a list of records, each record will be rendered at a new line:
![image](https://user-images.githubusercontent.com/100466382/200191652-eadbb736-17a2-4a1a-b13d-2e8876569532.png)
It seems like the following looks better:
![image](https://user-images.githubusercontent.com/100466382/200191674-92a4f825-1a1c-485b-baa3-3022cf36cebd.png)

**Comment by RDBModel** [comment]
I was able to fix such behavior by modifying `condeList` in Encode.elm:
```
encodeList : (a -> Encoder) -> EncoderState -> List a -> String
encodeList encode state l =
    let
        internalString val =
            (internalConvertToString
                { state | col = state.col + state.indent, prefix = True }
                << encode
            )
            val

        listElement : a -> String
        listElement val =
            "-" ++ (if internalString val |> String.startsWith "\n" then
                    internalString val |> String.trimLeft |> (++) " "
                else
                    internalString val)

        prefix : String
        prefix =
            if state.prefix then
                "\n"

            else
                ""

        indentAfter : String -> String
        indentAfter s =
            s ++ String.repeat state.col " "
    in
    List.map listElement l
        |> String.join (indentAfter "\n")
        |> String.append (indentAfter prefix)
```

**Comment by MaybeJustJames** [comment]
Seems reasonable. Would you like to submit a PR?

**Comment by MaybeJustJames** [comment]
@RDBModel I've just released `2.1.4` which fixes this. Thanks for your help :)

**Comment by RDBModel** [comment]
just checked the changes from 2.1.4 on https://rdbmodel.github.io/ and it is working well. Thanks!
